app/src/main/java/com/dodo/flashcards/presentation/register_screen/RegisterScreenViewModel.kt

```diff
@@ -0,0 +1,34 @@
+package com.dodo.flashcards.presentation.register_screen
+
+import androidx.lifecycle.SavedStateHandle
+import com.dodo.flashcards.architecture.BaseRoutingViewModel
+import com.dodo.flashcards.presentation.MainDestination
+import com.dodo.flashcards.presentation.register_screen.RegisterScreenViewEvent.*
```

Comment:
FYI `.*` here imports all `RegisterScreenViewEvent` so we don't need to do  `RegisterScreenViewEvent.ClickedRegister` which is messy